spec/unit/network/formats_spec.rb

```diff
@@ -130,6 +130,24 @@ def to_pson(*args)
     end
   end
 
+  describe "binary" do
+    before do
+      @binary = Puppet::Network::FormatHandler.format(:binary)
+    end
+
+    it "should have its mimetype set to application/octet-stream" do
+      @binary.mime.should == "application/octet-stream"
+    end
+
+    it "should fail if its multiple_render method is used" do
```

Comment:
I removed this test and restored the old `:raw` test and just updated it to support the `:binary` format instead.